### DISCUSSION TITLE
My other other account got suspended<title>

### DISCUSSION THREAD

**Discussion by JhunMalda** [community/community#128503] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi, my other account got suspended without any notice. Can anyone help me, please?
I really need this account.

Ticket ID: 2841164

Thank you.

**Comment by vdevcode** [comment]
I think the first thing you need to do is contact github to explain, maybe for some reason the system suspended your account or maybe there was a problem with the system.

**Comment by ebndev** [comment]
Hi @JhunMalda,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
